Scripts/generateDocumentation.sh

```diff
@@ -35,17 +35,7 @@ bash Scripts/deleteDuplicates.sh "$OUTPUT_DIRECTORY/$TARGET_DIRECTORY" "$TARGET_
 find "$OUTPUT_DIRECTORY/$TARGET_DIRECTORY" -type f -exec sed -i '' '1d' {} +
 
 if [[ "$TARGET" = "StreamChatUI" ]]; then
-   /bin/mv -v "$OUTPUT_DIRECTORY/$TARGET_DIRECTORY/"* "$OUTPUT_DIRECTORY/ui-components/"
-   bash Scripts/addImagesToDocumentation.sh "$OUTPUT_DIRECTORY/ui-components"
-else
-    # Right now, we want to add documentation only for controllers.
-    /bin/mv -v "$OUTPUT_DIRECTORY/$TARGET_DIRECTORY/Controllers/"* "$OUTPUT_DIRECTORY/controllers/"
+   bash Scripts/addImagesToDocumentation.sh "$OUTPUT_DIRECTORY/Sources/StreamChatUI"
```

Comment:
How much headache would it be for you to get rid of `Sources` make it `$OUTPUT_DIRECTORY/StreamChatUI` (same for `StreamChat`)?

Reply:
It would be a lot of breaking changes to the generation, I can take a look later when I have time,  but I agree, this is not ideal

Reply:
then lets skip it